TransferMethod/Tests/AddTransferMethodPresenterTests.swift

```diff
@@ -1,5 +1,7 @@
+import Common
 import Hippolyte
 import HyperwalletSDK
+import Insights
```

Comment:
remove this import

Reply:
Due to change in latest refactored code, these fixes has been done in following PR: https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/pull/167